README.md

```diff
@@ -1 +1,23 @@
-# platatus
+# Project Platatus
+
+*tomayto, tomahto*, or also **platform-status**
+
+## Goal
+
+Platatus gives developers a cross-platform and cross-browser roadmap for browser features, from standardization, development to release. Not only being a channel to raise awareness, it can also drive adoption and improve the vital feedback loop Firefox's platform needs to have with web developers. For most developers it will be a reference to discover and learn, but it has the potential to provide the entry points for radical participation on Mozilla's platform development.
+
+## Why
+
+Existing web compatibilty references, like [caniuse](http://caniuse.com/) focus on cross-browser compatibilty but don't cover un-released features and lag behind on features that already shipped. Major browser vendors ([Edge](https://dev.modern.ie/platform/status/), [Chrome](https://www.chromestatus.com/features), [Webkit](https://www.webkit.org/status.html)) are publicising their own platform status with references to their bug tracking and bug owners. Similar to Platatus' goal, Chrome tracks consensus/enthusiasm from other browsers using sources that range from bugs, to IRC logs to tweets.
```

Comment:
nit: Either remove the comma before "like caniuse" or add a comma at the end of it